Sources/SwiftCrossUI/Views/View.swift

```diff
@@ -18,7 +18,7 @@ public protocol View {
 
 extension View where Widget == GtkBox {
     public func asWidget(_ children: Content.Children) -> GtkBox {
-        let vStack = GtkBox(orientation: .vertical, spacing: 8)
+        let vStack = GtkBox(orientation: .vertical, spacing: 8).debugName(Self.self)
```

Comment:
This debugName thing is cool. Should it be added to the widgets of all views so that we don't need to keep adding it incrementally?

Reply:
Hmm wdym? It has to be called from the View to get its type